src/towncrier/test/test_build.py

```diff
@@ -671,3 +671,111 @@ def test_bullet_points_false(self):
+                    [tool.towncrier]
+                    package = "foo"
+                    title_format = "[{project_date}] CUSTOM RELEASE for {name} version {version}"
+                """))
+            os.mkdir("foo")
+            os.mkdir("foo/newsfragments")
+            with open("foo/newsfragments/123.feature", "w") as f:
+                f.write("Adds levitation")
+            # Towncrier ignores .rst extension
+            with open("foo/newsfragments/124.feature.rst", "w") as f:
+                f.write("Extends levitation")
+
+            result = runner.invoke(
+                _main,
+                [
+                    "--name",
+                    "FooBarBaz",
+                    "--version",
+                    "7.8.9",
+                    "--date",
+                    "20-01-2001",
+                    "--draft",
+                ],
+            )
+
+        expected_output = dedent("""\
+            Loading template...
+            Finding news fragments...
+            Rendering news fragments...
+            Draft only -- nothing has been written.
+            What is seen below is what would be written.
+
+            [20-01-2001] CUSTOM RELEASE for FooBarBaz version 7.8.9
+            =======================================================
+
+            Features
+            --------
+
+            - Adds levitation (#123)
+            - Extends levitation (#124)
+
+        """)
+
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(expected_output, result.output)
+
+    def test_title_format_false(self):
+        """
+        Setting the title format to false disables the explicit title.  This
+        would be used, for example, when the template creates the title itself.
+        """
+        runner = CliRunner()
+
+        with runner.isolated_filesystem():
+            with open("pyproject.toml", "w") as f:
+                f.write(dedent("""\
+                    [tool.towncrier]
+                    package = "foo"
+                    title_format = false
+                """))
+            os.mkdir("foo")
+            os.mkdir("foo/newsfragments")
+            # Towncrier ignores .rst extension
+            with open("foo/newsfragments/124.feature.rst", "w") as f:
+                f.write("Extends levitation")
+
+            result = runner.invoke(
+                _main,
+                [
+                    "--name",
+                    "FooBarBaz",
+                    "--version",
+                    "7.8.9",
+                    "--date",
+                    "20-01-2001",
+                    "--draft",
+                ],
+            )
+
+        expected_output = dedent("""\
+            Loading template...
+            Finding news fragments...
+            Rendering news fragments...
+            Draft only -- nothing has been written.
+            What is seen below is what would be written.
+
+            FooBarBaz 7.8.9 (20-01-2001)
```

Comment:
I missed this when reporting the fix worked for me... This is well incorrect, the title should not appear here as per `title_format = false` above?